pixl_dcmd/src/pixl_dcmd/_database.py

```diff
@@ -0,0 +1,66 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Interaction with the PIXL database."""
+from decouple import config
+
+from core.database import Image
+from sqlalchemy import URL, create_engine
+from sqlalchemy.orm import sessionmaker
+
+url = URL.create(
+    drivername="postgresql+psycopg2",
+    username=config("PIXL_DB_USER", default="None"),
+    password=config("PIXL_DB_PASSWORD", default="None"),
+    host=config("PIXL_DB_HOST", default="None"),
+    port=config("PIXL_DB_PORT", default=1),
+    database=config("PIXL_DB_NAME", default="None"),
+)
+
+engine = create_engine(url)
+
+
+def insert_new_uid_into_db_entity(existing_image: Image, hashed_value: str) -> Image:
```

Comment:
Maybe something like this
```suggestion
def add_hashed_identifier_and_save(existing_image: Image, hashed_value: str) -> Image:
```